thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/components/HoltWintersDetector.java

```diff
@@ -114,31 +120,27 @@ public void init(HoltWintersDetectorSpec spec, InputDataFetcher dataFetcher) {
     this.sensitivity = spec.getSensitivity();
     this.monitoringGranularity = spec.getMonitoringGranularity();
 
-    if (this.monitoringGranularity.equals("1_MONTHS")) {
+    if (this.monitoringGranularity.endsWith("MONTHS") || this.monitoringGranularity.endsWith("WEEKS")) {
```

Comment:
Should we define `"MONTHS"` and `"WEEKS"` as constants?

Reply:
Updated